behave/formatter/plain.py

```diff
@@ -99,8 +100,9 @@ def result(self, result):
 
         status_text = result.status.name
         if self.show_timings:
```

Comment:
I would prefer not to change the way results are displayed. However, we could exploit 'show_timings' to have another value that would also print the starting time. Sth. like:
if show_timings == 2:
 ...
WDYT?